lib/Bio/EnsEMBL/DataCheck/Checks/ChromosomesAnnotated.pm

```diff
@@ -0,0 +1,73 @@
+=head1 LICENSE
+
+Copyright [2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::ChromosomesAnnotated;
+
+use warnings;
+use strict;
+
+use Moose;
+use Test::More;
+use Bio::EnsEMBL::DataCheck::Utils qw/sql_count/;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {
+  NAME           => 'ChromosomesAnnotated',
+  DESCRIPTION    => 'Check whether chromosomal seq_regions have appropriate attribute.',
+  GROUPS         => ['assembly'],
+  DB_TYPES       => ['core'],
+  TABLES         => ['attrib_type', 'coord_system', 'seq_region', 'seq_region_attrib']
+};
+
+sub skip_tests {
+  my ($self) = @_;
+  my $species_id = $self->dba->species_id;
```

Comment:
Maybe consider using the DC species method just like in the AttribValues DC?

Reply:
AttribValues is restricting the application of the datacheck to a subset of species. In this case I use species_id to make sure the query (and subsequent skipping) is restricted to one species at a time for collection dbs.